integration/basic_test.go

```diff
@@ -106,16 +106,40 @@ func TestInsertFind(t *testing.T) {
 //nolint:paralleltest // we test a global list of databases
 func TestFindCommentMethod(t *testing.T) {
 	ctx, collection := setup.Setup(t, shareddata.Scalars)
-	name := collection.Name()
+	collName := collection.Name()
```

Comment:
why did you change it?

Reply:
It's not ready for review yet. Sorry for false alarm, I wanted to create a draft :)

Reply:
Also I thought it would be more readable but if we prefer to have an old name I'll keep it in mind